src/test/ui/feature-gates/feature-gate-profiler-runtime.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0658]: the `#[profiler_runtime]` attribute is used to identify the `profiler_builtins` crate which contains the profiler runtime and will never be stable
+error[E0658]: the `#[profiler_runtime]` attribute is used to identify the crate which contains the profiler runtime and will never be stable
```

Comment:
Is this attribute still used anywhere? Can we remove it altogether?

Reply:
There is an unstable option "-Zprofiler-runtime=<crate_name>" for users to inject their custom profiler runtime. The `profiler_runtime` attribute is the profiler-runtime-crate marker. So if the attribute is removed, `-Zprofiler-runtime` will be useless.